featuretools/entityset/entityset.py

```diff
@@ -715,6 +715,13 @@ def normalize_entity(self, base_entity_id, new_entity_id, index,
             if v == index:
                 raise ValueError("Not copying {} as both index and variable".format(v))
                 break
+
+        if isinstance(make_time_index, str):
+            if make_time_index not in base_entity.df.columns:
+                raise ValueError("'make_time_index' must be a variable in the base frame")
+            elif make_time_index not in additional_variables + copy_variables + [index]:
+                raise ValueError("'make_time_index' must be a variable specified in 'additional_variables' or 'copy_variables'")
```

Comment:
Should we also say or `'additional_variables' or 'copy_variables' or 'index'` in the error message? Since we are checking in that field as well.